esp/src/eclwatch/LogWidget.js

```diff
@@ -232,7 +232,7 @@ define([
         refreshGrid: function (clearSelection) {
             this.rawText.setText(this.i18n.loadingMessage);
             var filter = lang.mixin(this.getFilter(), {
-                Name: this.params.getNetaddress ? "//" + this.params.getNetaddress() + this.params.getLogDirectory() + "/" + this.logTargetSelect.get("value") : "//" + this.params.params.Netaddress + this.params.LogDirectory + "/" + this.logTargetSelect.get("value"),
+                Name: this.params.getNetaddress ? "//" + this.params.getNetaddress() + this.params.getLogDirectory() + "/" + this.logTargetSelect.get("value") : "//" + this.params.NetAddress + this.params.LogDirectory + "/" + this.logTargetSelect.get("value"),
```

Comment:
That params.params object does exist, it just doesn't get serialized correctly evidently. But params.NetAddress is always available as well. The other files changed were necessary for hitting these "Open in new page" links from the v9 UI.